robottelo/cli/factory.py

```diff
@@ -3586,36 +3586,6 @@ def setup_virtual_machine(
         vm.install_katello_agent()
 
 
-def extract_capsule_satellite_installer_command(text):
```

Comment:
this is not a factory method, moving to helpers.

Reply:
This function is used only here we can rename it to be private (with underscore)

Reply:
it is also used in the tes_capsule_installer test case directly, i would prefer to keep it in helpers if it is okay with you.